Pod/Classes/PINAnimatedImage.m

```diff
@@ -127,7 +127,7 @@ void releaseData(void *data, const void *imageData, size_t size)
   CFRelease(data);
 }
 
-- (CGImageRef)imageAtIndex:(NSUInteger)index inSharedImageFiles:(NSArray <PINSharedAnimatedImageFile *>*)imageFiles width:(UInt32)width height:(UInt32)height bitsPerPixel:(UInt32)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
+- (CGImageRef)imageForIndex:(NSUInteger)index inSharedImageFiles:(NSArray <PINSharedAnimatedImageFile *>*)imageFiles width:(UInt32)width height:(UInt32)height bitsPerPixel:(UInt32)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
```

Comment:
Curious why did you change it to `imageForIndex:...`? `objectAtIndex:...` is I think the common way in Apple APIs